### PR TITLE
Add more information about slow bluetooth connections

### PR DESCRIPTION
**Pull request by bdraco**
## Proposed change

Add more information about slow bluetooth connections and what to do to resolve the performance issues.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [ ] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/bluetooth.markdown [hunk 1]
```diff
@@ -251,6 +255,19 @@ The following remote adapters are supported:
 
 ## Troubleshooting
 
+### Improving connection times
+
+The connection time and performance varies greatly based on the bluetooth adapter. Below adapters are listed from best performing to worst performing:
```
Comment (bdraco) [inline source/_integrations/bluetooth.markdown]:
```suggestion
The connection time and performance vary greatly based on the Bluetooth adapter. The below adapters are listed from best performing to worst performing:
```

File: source/_integrations/bluetooth.markdown [hunk 2]
```diff
@@ -251,6 +255,19 @@ The following remote adapters are supported:
 
 ## Troubleshooting
 
+### Improving connection times
+
+The connection time and performance vary greatly based on the Bluetooth adapter. The below adapters are listed from best performing to worst performing:
+
+- Bluetooth proxies running ESPHome 2023.6.0 or later with passive scanning
+- USB High performance adapter with passive scanning
```
Comment (bdraco) [inline source/_integrations/bluetooth.markdown]:
```suggestion
- [USB High performance adapter](#known-working-high-performance-adapters) with passive scanning
```

File: source/_integrations/bluetooth.markdown [hunk 3]
```diff
@@ -251,6 +255,19 @@ The following remote adapters are supported:
...
+- Bluetooth proxies running ESPHome 2023.6.0 or later with active scanning
+- USB High performance adapter with active scanning
```
Comment (bdraco) [inline source/_integrations/bluetooth.markdown]:
```suggestion
- [USB High performance adapter](#known-working-high-performance-adapters) with active scanning
```

File: source/_integrations/bluetooth.markdown [hunk 4]
```diff
@@ -251,6 +255,19 @@ The following remote adapters are supported:
 
 ## Troubleshooting
 
+### Improving connection times
+
+The connection time and performance vary greatly based on the Bluetooth adapter. The below adapters are listed from best performing to worst performing:
+
+- Bluetooth proxies running ESPHome 2023.6.0 or later with passive scanning
+- [USB High performance adapter](#known-working-high-performance-adapters) with passive scanning
```
Comment (bdraco) [inline source/_integrations/bluetooth.markdown]:
```suggestion
- [USB High performance adapter](#known-working-high-performance-adapters) with [passive scanning](#passive-scanning)
```

File: source/_integrations/bluetooth.markdown [hunk 5]
```diff
@@ -251,6 +255,19 @@ The following remote adapters are supported:
...
+- Bluetooth proxies running ESPHome 2023.6.0 or later with active scanning
+- [USB High performance adapter](#known-working-high-performance-adapters) with active scanning
+- Onboard high performance adapter with passive scanning
```
Comment (bdraco) [inline source/_integrations/bluetooth.markdown]:
```suggestion
- Onboard high performance adapter with [passive scanning](#passive-scanning)
```

File: source/_integrations/bluetooth.markdown [hunk 6]
```diff
@@ -251,6 +255,19 @@ The following remote adapters are supported:
...
+- Onboard high performance adapter with active scanning
+- Known working adapters with passive scanning
```
Comment (bdraco) [inline source/_integrations/bluetooth.markdown]:
```suggestion
- Known working adapters with [passive scanning](#passive-scanning)
```

File: source/_integrations/bluetooth.markdown [hunk 7]
```diff
@@ -251,6 +255,19 @@ The following remote adapters are supported:
...
+The connection time and performance vary greatly based on the Bluetooth adapter. The below adapters are listed from best performing to worst performing:
+
+- Bluetooth proxies running ESPHome 2023.6.0 or later with passive scanning
+- [USB High performance adapter](#known-working-high-performance-adapters) with [passive scanning](#passive-scanning)
+- Bluetooth proxies running ESPHome 2023.6.0 or later with active scanning
+- [USB High performance adapter](#known-working-high-performance-adapters) with active scanning
+- Onboard high performance adapter with [passive scanning](#passive-scanning)
+- Onboard high performance adapter with active scanning
+- Known working adapters with [passive scanning](#passive-scanning)
+- Known working adapters with active scanning
```
Comment (bdraco) [inline source/_integrations/bluetooth.markdown]:
```suggestion
- [Known working adapters](#known-working-adapters) with [passive scanning](#passive-scanning)
- [Known working adapters[(#known-working-adapters) with active scanning
```

File: source/_integrations/bluetooth.markdown [hunk 8]
```diff
@@ -251,6 +255,19 @@ The following remote adapters are supported:
...
+### Improving connection times
+
+The connection time and performance vary greatly based on the Bluetooth adapter. The below adapters are listed from best performing to worst performing:
+
+- Bluetooth proxies running ESPHome 2023.6.0 or later with passive scanning
+- [USB High performance adapter](#known-working-high-performance-adapters) with [passive scanning](#passive-scanning)
+- Bluetooth proxies running ESPHome 2023.6.0 or later with active scanning
+- [USB High performance adapter](#known-working-high-performance-adapters) with active scanning
+- Onboard high performance adapter with [passive scanning](#passive-scanning)
+- Onboard high performance adapter with active scanning
```
Comment (bdraco) [inline source/_integrations/bluetooth.markdown]:
```suggestion
- [Onboard high performance adapter](#cypress-based-adapters) with [passive scanning](#passive-scanning)
- [Onboard high performance adapter](#cypress-based-adapters) with active scanning
```

File: source/_integrations/bluetooth.markdown [hunk 9]
```diff
@@ -251,6 +255,19 @@ The following remote adapters are supported:
 
 ## Troubleshooting
 
+### Improving connection times
+
+The connection time and performance vary greatly based on the Bluetooth adapter. The below adapters are listed from best performing to worst performing:
+
+- Bluetooth proxies running ESPHome 2023.6.0 or later with passive scanning
+- [USB High performance adapter](#known-working-high-performance-adapters) with [passive scanning](#passive-scanning)
+- Bluetooth proxies running ESPHome 2023.6.0 or later with active scanning
```
Comment (bdraco) [inline source/_integrations/bluetooth.markdown]:
```suggestion
- [Bluetooth proxies](#remote-adapters-bluetooth-proxies) running ESPHome 2023.6.0 or later with [passive scanning](https://esphome.io/components/esp32_ble_tracker.html#configuration-variables)
- [USB High performance adapter](#known-working-high-performance-adapters) with [passive scanning](#passive-scanning)
- [Bluetooth proxies](#remote-adapters-bluetooth-proxies) running ESPHome 2023.6.0 or later with [active scanning](https://esphome.io/components/esp32_ble_tracker.html#configuration-variables)
```

File: source/_integrations/bluetooth.markdown [hunk 10]
```diff
@@ -163,6 +163,10 @@ Performance testing used the following hardware:
 
 ### Known working adapters
 
+<div class='note'>
+Known working adapters list adapter that do not meet high performance requirements, but will generally work. These adapters vary widely in performance, and may take as long as thirty seconds or more to establish a connection.
```
Comment (bdraco) [inline source/_integrations/bluetooth.markdown]:
```suggestion
Known working adapters list adapters that do not meet high-performance requirements but will generally work. These adapters vary widely in performance and may take as long as thirty seconds or more to establish a connection.
```

File: source/_integrations/bluetooth.markdown [hunk 11]
```diff
@@ -251,6 +255,19 @@ The following remote adapters are supported:
 
 ## Troubleshooting
 
+### Improving connection times
+
+The connection time and performance vary greatly based on the Bluetooth adapter. The below adapters are listed from best performing to worst performing:
```
Comment (bdraco) [inline source/_integrations/bluetooth.markdown]:
```suggestion
The connection time and performance vary greatly based on the Bluetooth adapter. The below adapters are listed from best-performing to worst-performing:
```

File: source/_integrations/bluetooth.markdown [hunk 12]
```diff
@@ -251,6 +255,19 @@ The following remote adapters are supported:
...
+The connection time and performance vary greatly based on the Bluetooth adapter. The below adapters are listed from best-performing to worst-performing:
+
+- [Bluetooth proxies](#remote-adapters-bluetooth-proxies) running ESPHome 2023.6.0 or later with [passive scanning](https://esphome.io/components/esp32_ble_tracker.html#configuration-variables)
+- [USB High performance adapter](#known-working-high-performance-adapters) with [passive scanning](#passive-scanning)
+- [Bluetooth proxies](#remote-adapters-bluetooth-proxies) running ESPHome 2023.6.0 or later with [active scanning](https://esphome.io/components/esp32_ble_tracker.html#configuration-variables)
+- [USB High performance adapter](#known-working-high-performance-adapters) with active scanning
+- [Onboard high performance adapter](#cypress-based-adapters) with [passive scanning](#passive-scanning)
+- [Onboard high performance adapter](#cypress-based-adapters) with active scanning
+- [Known working adapters](#known-working-adapters) with [passive scanning](#passive-scanning)
+- [Known working adapters[(#known-working-adapters) with active scanning
```
Comment (bdraco) [inline source/_integrations/bluetooth.markdown]:
```suggestion
- [Known working adapters](#known-working-adapters) with active scanning
```

File: source/_integrations/bluetooth.markdown [hunk 13]
```diff
@@ -251,6 +255,19 @@ The following remote adapters are supported:
 
 ## Troubleshooting
 
+### Improving connection times
+
+The connection time and performance vary greatly based on the Bluetooth adapter. The below adapters are listed from best-performing to worst-performing:
```
Comment (bdraco) [inline source/_integrations/bluetooth.markdown]:
```suggestion
The connection time and performance vary greatly based on the Bluetooth adapter and interference. The below adapters are listed from best-performing to worst-performing:
```

File: source/_integrations/bluetooth.markdown [hunk 14]
```diff
@@ -251,6 +255,19 @@ The following remote adapters are supported:
...
+
+- [Bluetooth proxies](#remote-adapters-bluetooth-proxies) running ESPHome 2023.6.0 or later with [passive scanning](https://esphome.io/components/esp32_ble_tracker.html#configuration-variables)
```
Comment (bdraco) [inline source/_integrations/bluetooth.markdown]:
```suggestion
- [Ethernet-connected Bluetooth proxies](#remote-adapters-bluetooth-proxies) running ESPHome 2023.6.0 or later with [passive scanning](https://esphome.io/components/esp32_ble_tracker.html#configuration-variables)
```

File: source/_integrations/bluetooth.markdown [hunk 15]
```diff
@@ -251,6 +255,19 @@ The following remote adapters are supported:
...
+- [USB High performance adapter](#known-working-high-performance-adapters) with [passive scanning](#passive-scanning)
+- [Bluetooth proxies](#remote-adapters-bluetooth-proxies) running ESPHome 2023.6.0 or later with [active scanning](https://esphome.io/components/esp32_ble_tracker.html#configuration-variables)
```
Comment (bdraco) [inline source/_integrations/bluetooth.markdown]:
```suggestion
- [Ethernet-connected Bluetooth proxies](#remote-adapters-bluetooth-proxies) running ESPHome 2023.6.0 or later with [active scanning](https://esphome.io/components/esp32_ble_tracker.html#configuration-variables)
```

File: source/_integrations/bluetooth.markdown [hunk 16]
```diff
@@ -251,6 +255,19 @@ The following remote adapters are supported:
...
 ## Troubleshooting
 
+### Improving connection times
+
+The connection time and performance vary greatly based on the Bluetooth adapter and interference. The below adapters are listed from best-performing to worst-performing:
+
+- [Ethernet-connected Bluetooth proxies](#remote-adapters-bluetooth-proxies) running ESPHome 2023.6.0 or later with [passive scanning](https://esphome.io/components/esp32_ble_tracker.html#configuration-variables)
+- [USB High performance adapter](#known-working-high-performance-adapters) with [passive scanning](#passive-scanning)
+- [Ethernet-connected Bluetooth proxies](#remote-adapters-bluetooth-proxies) running ESPHome 2023.6.0 or later with [active scanning](https://esphome.io/components/esp32_ble_tracker.html#configuration-variables)
+- [USB High performance adapter](#known-working-high-performance-adapters) with active scanning
```
Comment (bdraco) [inline source/_integrations/bluetooth.markdown]:
```suggestion
- [Ethernet-connected Bluetooth proxies](#remote-adapters-bluetooth-proxies) running ESPHome 2023.6.0 or later with [passive scanning](https://esphome.io/components/esp32_ble_tracker.html#configuration-variables)
- [USB High performance adapter](#known-working-high-performance-adapters) with [passive scanning](#passive-scanning)
- [WiFi-connected Bluetooth proxies](#remote-adapters-bluetooth-proxies) running ESPHome 2023.6.0 or later with [passive scanning](https://esphome.io/components/esp32_ble_tracker.html#configuration-variables)
- [Ethernet-connected Bluetooth proxies](#remote-adapters-bluetooth-proxies) running ESPHome 2023.6.0 or later with [active scanning](https://esphome.io/components/esp32_ble_tracker.html#configuration-variables)
- [USB High performance adapter](#known-working-high-performance-adapters) with active scanning
- [WiFi-connected Bluetooth proxies](#remote-adapters-bluetooth-proxies) running ESPHome 2023.6.0 or later with [active scanning](https://esphome.io/components/esp32_ble_tracker.html#configuration-variables)
```

File: source/_integrations/bluetooth.markdown [hunk 17]
```diff
@@ -251,6 +255,21 @@ The following remote adapters are supported:
 
 ## Troubleshooting
 
+### Improving connection times
+
+The connection time and performance vary greatly based on the Bluetooth adapter and interference. The below adapters are listed from best-performing to worst-performing:
+
+- [Ethernet-connected Bluetooth proxies](#remote-adapters-bluetooth-proxies) running ESPHome 2023.6.0 or later with [passive scanning](https://esphome.io/components/esp32_ble_tracker.html#configuration-variables)
+- [USB High performance adapter](#known-working-high-performance-adapters) with [passive scanning](#passive-scanning)
+- [WiFi-connected Bluetooth proxies](#remote-adapters-bluetooth-proxies) running ESPHome 2023.6.0 or later with [passive scanning](https://esphome.io/components/esp32_ble_tracker.html#configuration-variables)
```
Comment (c0ffeeca7) [inline source/_integrations/bluetooth.markdown]:
```suggestion
- [Wi-Fi-connected Bluetooth proxies](#remote-adapters-bluetooth-proxies) running ESPHome 2023.6.0 or later with [passive scanning](https://esphome.io/components/esp32_ble_tracker.html#configuration-variables)
```

File: source/_integrations/bluetooth.markdown [hunk 18]
```diff
@@ -251,6 +255,21 @@ The following remote adapters are supported:
...
+- [Ethernet-connected Bluetooth proxies](#remote-adapters-bluetooth-proxies) running ESPHome 2023.6.0 or later with [active scanning](https://esphome.io/components/esp32_ble_tracker.html#configuration-variables)
+- [USB High performance adapter](#known-working-high-performance-adapters) with active scanning
+- [WiFi-connected Bluetooth proxies](#remote-adapters-bluetooth-proxies) running ESPHome 2023.6.0 or later with [active scanning](https://esphome.io/components/esp32_ble_tracker.html#configuration-variables)
```
Comment (c0ffeeca7) [inline source/_integrations/bluetooth.markdown]:
```suggestion
- [Wi-Fi-connected Bluetooth proxies](#remote-adapters-bluetooth-proxies) running ESPHome 2023.6.0 or later with [active scanning](https://esphome.io/components/esp32_ble_tracker.html#configuration-variables)
```

**Review by c0ffeeca7 — CHANGES_REQUESTED** [review]

File: source/_integrations/bluetooth.markdown [hunk 19]
```diff
@@ -251,6 +255,21 @@ The following remote adapters are supported:
 
 ## Troubleshooting
 
+### Improving connection times
```
Comment (c0ffeeca7) [inline source/_integrations/bluetooth.markdown:258]: The title "Improving connection times" and the PR message suggest that there are instructions on how to improve connection times. 
So the measure to take is to use a different device, one that is higher up in the list?

**Comment by bdraco** [comment]
changes LGTM

File: source/_integrations/bluetooth.markdown [hunk 20]
```diff
@@ -163,6 +163,10 @@ Performance testing used the following hardware:
 
 ### Known working adapters
 
+<div class='note'>
+Known working adapters list adapters that do not meet high-performance requirements but will generally work. These adapters vary widely in performance and may take as long as thirty seconds or more to establish a connection.
```
Comment (bdraco) [inline source/_integrations/bluetooth.markdown]:
```suggestion
Known working adapters list adapters that do not meet high-performance requirements but will generally work. These adapters vary widely in performance and may take as long as thirty seconds or more to establish a connection. These adapters may also miss advertisements such as button presses or temperature updates.
```

**Review by frenck — APPROVED** [review]
Thanks, @bdraco 👍

../Frenck

**Review by c0ffeeca7 — APPROVED** [review]

**Comment by bdraco** [comment]
thanks!

**Pull request merged**